maxim/tests/test_groq.py

```
@@ -0,0 +1,445 @@
+            print("Response:", response.choices[0].message.content)
+        except Exception as e:
+            self.skipTest(f"Error: {e}")
+
+    def test_non_streaming_with_system_message(self):
+        """Test non-streaming chat completion with system message"""
+        try:
+            response = self.client.chat.completions.create(
+                model="llama-3.1-8b-instant",
+                messages=[
+                    {"role": "system", "content": "You are a helpful travel assistant. Always provide concise, practical advice."},
+                    {"role": "user", "content": "What's the best way to get around San Francisco?"}
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)